website/docs/reference/resource-configs/grants.md

```diff
@@ -249,7 +249,20 @@ models:
 
 <div warehouse="Redshift">
 
-* Granting to / revoking from is only fully supported for Redshift users (not groups or roles).
+For Redshift, you need to explicitly grant roles and groups access to your models, whether to single users, groups, or roles, and you can revoke access as well.
```

Comment:
Would this be better suited here with a link from the note https://docs.getdbt.com/reference/resource-configs/redshift-configs or maybe at least also there?